src/styles/blocks/item.scss

```diff
@@ -0,0 +1,80 @@
+.item {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: $items-width;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    margin-bottom: 40px;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    margin: 0;
+    margin-bottom: 4px;
+
+    font-size: $small-font-size;
+    font-weight: $medium-font-weight;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+
```

Comment:
Isn't it better to use margin-bottom here?

Reply:
> upvote

I had to reset the extra margin for them!